### PR TITLE
I23 create a nav bar component for mobile

### LINKED ISSUE #23: Create a nav bar component for mobile
## Basic Information
The navbar should sit at the bottom of the screen.
There should be 4 tabs or pages:
- [ ] Home
- [ ] Contact
- [ ] Visit
- [ ] Incident reports

## Other Information
Potential scopes:

### PR DESCRIPTION
**Pull request by pavo-etc**
## Change Summary
Created bottom navigation bar and with `poops-red` color.

### Change Form

- [x] The pull request title has an issue number
- [x] The change works by "Smoke testing" or quick testing
- [ ] The change has tests - I am unsure how tests should be written for a simple component like this 
- [ ] The change has documentation - Same as tests

## Other Information

This change adds the following dependencies:
```
"@mdi/font": "^6.7.96",
"@mdi/js": "^6.7.96",
"@mdi/react": "^1.6.0"
```
These are used for the navigation icons.  Were intended to be placeholders though they work quite well.

This uses the same shade of red from the [POOPs website](https://www.poopswa.org.au/).  This is added as `poops-red` to Tailwind and `var(--poops-red)` in `global.css`, and has the hex code `#ce283d`.

# Related Issue

- Resolve #23

### CODE REVIEW

**Comment by scratchclaggy** [comment]
Just a suggestion: but seeming as we're using tailwind for our UI framework, we could also use [heroicons](https://github.com/tailwindlabs/heroicons) for our iconography. It uses the same interface as regular text so it can take all of our tailwind classes.

**Comment by ChantelleKerr** [comment]
> 

The icons are adorable, however mdi has a wider range of icons.

**Comment by pavo-etc** [comment]
Have replaced material design icons with heroicons, and have added a tab indicator as in the wireframe mockup.

**Comment by skyheat** [comment]
Hey guys, looks great so far! Just added a WIP tag to the issue name, remove it when you're ready for it to be reviewed and merged :)

It was our bad, we should've mentioned it in the good practices

**Comment by ChantelleKerr** [comment]
Note: I've added a temp dog icon - This should probably be replaced with poops icon if they have one

File: src/components/navBar/NavBar.tsx [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+    {
+      name: 'Incidents',
+      href: '#',
+      icon: <ExclamationCircleIcon className={iconClasses} />
+    }
+  ]
+
+  const navLinks = linkDetails.map((link) => {
+    return (
+      <NavLink
```
Comment (skyheat) [inline src/components/navBar/NavBar.tsx:37]: Nice and clean

File: src/pages/_app.tsx [hunk 2]
```diff
@@ -1,5 +1,7 @@
 import { AppProps } from 'next/app'
 
+import NavBar from '@/components/navBar/NavBar'
+
 import '@/styles/globals.css'
 // !STARTERCONF This is for demo purposes, remove @/styles/colors.css import immediately
 import '@/styles/colors.css'
```
Comment (skyheat) [inline src/pages/_app.tsx:7]: This should be deleted

File: src/components/navBar/NavLink.tsx [hunk 3]
```diff
@@ -0,0 +1,30 @@
...
+  icon,
+  currentPage,
+  setCurrentPage
+}: NavLinkProps) {
+  const tabClasses =
+    'inline-block w-full justify-center pt-2 pb-1 text-center hover:text-poops-red'
+  const currentTabClasses = tabClasses + ' border-t-2 border-t-poops-red'
+
+  return (
+    <a
```
Comment (skyheat) [inline src/components/navBar/NavLink.tsx:21]: Should be using Next link as it is internal routing, the only time you wouldn't want to be using it is when opening a new tab.

**Review by skyheat — CHANGES_REQUESTED** [review]
Good and clean code. Just a few minor changes. 

Good stuff guys 😄

File: src/components/navBar/NavLink.tsx (hunk 3 above)
Comment (ChantelleKerr) [reply src/components/navBar/NavLink.tsx]: I've updated it so that it uses Next Link, let me know if you have any more problems :)

**Review by skyheat — APPROVED** [review]
Looks great. Good job!

**Pull request merged**